topics/sdn/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: software-defined-networking, software-defined-network
+display_name: Software-defined networking
+related: networking, openflow, nfv, sd-wan
+short_description: Software-defined networking (SDN) is an architecture for programmable, centrally managed networks.
+topic: sdn
+wikipedia_url: https://en.wikipedia.org/wiki/Software-defined_networking
+---
+Software-defined networking (SDN) separates network control from network data forwarding, allowing networks to be programmed and centrally managed with standard protocols, enabling dynamic, vendor-neutral network management. SDN is an enabler of network virtualization, or the ability to run multiple virtual network topologies on a shared physical network.
```

Comment:
The Travis CI build is failing since it expects a comma before the "and" here, but this "and" is used as part of an item in a list and therefore should not have a comma.

Looking at it again, the sentence in question is ambiguous at best, dare I say grammatically incorrect. Let me explain.

Your sentence reads like a list, starting with `, allowing`, followed by `, enabling`, yet without the final `, and <verb>ing`. I'm sure that's not what you intended, but this is indeed a valid way to read it. What is "enabling" things  to happen if it isn't a part of the list that refers to SDN? This sentence is ambiguous at best.

One solution would be to add "thereby" before "enabling". This implies that the previous section is doing the enabling, and that the word `enabling` isn't a part of a list of `-ing` verbs. Another solution would be to simplify the description and make it overall easier to read (likely fixing the comma issue at the same time).

Reply:
That is a pretty terrible sentence. I'll try breaking it up momentarily. Thanks for the critique!